.github/workflows/gate.yaml

```diff
@@ -131,6 +131,14 @@ jobs:
       - name: Test
         run: ctest -j2 --output-on-failure -E unique-stigids
         working-directory: ./build
+      - name: "Set git safe directory, ref: https://github.com/actions/checkout/issues/760"
+        run: git config --global --add safe.directory "$GITHUB_WORKSPACE"
+      - name: Upload coverage to Code Climate  # Requires: git package
+        uses: paambaati/codeclimate-action@v3.0.0
+        env:
+          CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
```

Comment:
Yes, it may be that secrets aren't exposed to PRs. It is a question whether our reporter ID is a secret, and if so, what to do with it. I will continue looking into it.